docs/source/federation/implementing.md

```diff
@@ -97,38 +112,78 @@ server.listen(4001).then(({ url }) => {
     console.log(`🚀 Server ready at ${url}`);
 });
 ```
-> Note: we're now providing a `schema` to the `ApolloServer` constructor, rather than `typeDefs` and `resolvers`.
 
-Now the service is running as a federated service and is ready to be composed into an overall federated graph!
+The server is now ready to be added to a federated data graph!
+
+Here's the complete code sample for our federation-ready server:
+
+```js:title=index.js
+const { ApolloServer, gql } = require('apollo-server');
+const { buildFederatedSchema } = require('@apollo/federation');
+
+const typeDefs = gql`
+  type Query {
+    me: User
+  }
+
+  type User @key(fields: "id") {
+    id: ID!
+    username: String
+  }
+`;
+
+const resolvers = {
+  Query: {
+    me() {
+      return { id: "1", username: "@ava" }
+    }
+  },
+  User: {
+    __resolveReference(user, { fetchUserById }){
+      return fetchUserById(user.id)
+    }
+  }
+}
+
+const server = new ApolloServer({
+  schema: buildFederatedSchema([{ typeDefs, resolvers }])
+});
+
+server.listen(4001).then(({ url }) => {
+    console.log(`🚀 Server ready at ${url}`);
+});
+```
 
 ## Running a gateway
 
-Now that we have a federation-ready service, we can get started with building our gateway. First let's install the gateway package and Apollo Server:
+Now that we have a federation-ready service, we can build our gateway as a separate
+service. First, let's install Apollo Server and the `@apollo/gateway` package:
```

Comment:
hm. Could we avoid using the term `service` here? If we want a noun, I would say "server"